azdev/operations/tests/__init__.py

```diff
@@ -42,8 +42,8 @@ def run_tests(tests, xml_path=None, discover=False, in_series=False,
     heading('Run Tests')
 
     original_profile = _get_profile(profile)
-    if not profile:
-        profile = original_profile
+    profile = original_profile if profile is None else profile
```

Comment:
I think this line is unnecessary because _get_profile() already sets profile to original_profile when profile is None.

Reply:
the name `_get_profile()` confuse me. It did set and get thing but it's named get. :(